src/server/server.ts

```diff
@@ -15,15 +15,15 @@ import { ProxyMapping } from '../types/general'
 
 const cyan = '\x1b[36m\u001b[1m%s\x1b[0m'
 const red = '\x1b[31m\u001b[1m%s\x1b[0m'
+const warningMsg = 'WARNING! You have to setup ADMIN password first before to run the app.'
```

Comment:
this is an error. `ERROR: App cannot be started because you must set an ADMIN password in the environment variable when running this app. Visit https://github.com/garageScript/myproxy` <- should link to the correct section in readme.